CONTRIBUTING.md

```diff
@@ -2,35 +2,41 @@
 
 Hi! We're really excited that you are interested in contributing to ACPaaS UI. Before submitting your contribution though, please make sure to take a moment and read through the following guidelines.
 
-- [Code of Conduct](CODE_OF_CONDUCT.md)
-- [Issue Reporting Guidelines](#issue-reporting-guidelines)
-- [Pull Request Guidelines](#pull-request-guidelines)
-- [Development Setup](#development-setup)
-- [Project Structure](#project-structure)
+- [ACPaaS UI Contributing Guide](#acpaas-ui-contributing-guide)
+  - [Issue Reporting Guidelines](#issue-reporting-guidelines)
+  - [Pull Request Guidelines](#pull-request-guidelines)
+  - [Development Setup](#development-setup)
+    - [Committing Changes](#committing-changes)
+    - [Commonly used NPM scripts](#commonly-used-npm-scripts)
+  - [Releases](#releases)
+  - [Credits](#credits)
 
 ## Issue Reporting Guidelines
 
-- Always use [GitHub Issues][github-issues] to create new issues.
+- Use the [#acpaas-ui-react channel][acpaas-ui-react-slack] on slack to ask questions or discuss changes to the code itself.
+- Use [GitHub Issues][github-issues] to report bugs, request features, ask policy questions or propose policy changes.
 
-## Pull Request Guidelines
-
-- The `master` branch is basically just a snapshot of the latest stable release. All development should be done in dedicated branches. **Do not submit PRs against the `master` branch.**
+  > Note: policy questions are about the way ACPaaS UI is developed and released, or about its architecture. We use github issues to have a public archive of these discussions.
 
-- Checkout a topic branch from the relevant branch, e.g. `dev`, and merge back against that branch.
+## Pull Request Guidelines
 
-- Work in the `src` folder and **DO NOT** check in `dist` in the commits.
+- The `master` branch is where we ship releases from. Always send PR's to `master`.
 
-- It's OK to have multiple small commits as you work on the PR - we will let GitHub automatically squash it before merging.
+- Commit your changes to a topic branch with a name matching these examples:
+  - `feature/more-cowbell`
+  - `fix/42-broken-cowbell` (the issue id is in the branch name)
+  - `docs/document-cowbell`
+  
+- Work in the `packages` folder and **DO NOT** check in the top-level `dist` or `styleguide` folders in the commits.
```

Comment:
Ik denk wel dat het een meerwaarde kan zijn om de styleguide mee te commiten. Op dit moment genereren we die nog niet automatisch... 

We doen dit trouwens nu ook al onder /docs

Reply:
Hey @Jan-Bart, volgens mij wordt die wel al automatisch gegenereerd: https://digipolisantwerp.github.io/acpaas-ui_react/

Reply:
Ah, inderdaad, maar ik bedoelde on build. Je moet hem nog altijd mee committen als je hem in github wil zien => https://digipolisantwerp.github.io/acpaas-ui_react/

Maar je hebt wel gelijk :D